src/xunit.netcore.extensions/Attributes/ActiveIssueAttribute.cs

```diff
@@ -16,5 +16,7 @@ public class ActiveIssueAttribute : Attribute, ITraitAttribute
     {
         public ActiveIssueAttribute(int issueNumber, TestPlatforms platforms = TestPlatforms.Any) { }
         public ActiveIssueAttribute(string issue, TestPlatforms platforms = TestPlatforms.Any) { }
+        public ActiveIssueAttribute(int issueNumber, TargetFrameworkMonikers framework) { }
```

Comment:
Why not all TestPlatforms and TargetFrameworkMonikers to both be passed? I would just add TargetFrameworkMonikers as another optional parameter and default it to 0 (which is an no value).